app/models/admin_setting.rb

```diff
@@ -60,7 +60,7 @@ def self.check_queue
       if Date.today >= self.invite_from_queue_at.to_date
         new_date = Time.now + self.invite_from_queue_frequency.days
         self.first.update_attribute(:invite_from_queue_at, new_date)
-        InviteRequest.invite
+        InviteFromQueueJob.perform_now(count: invite_from_queue_number)
```

Comment:
Did you also find that ActiveJob doesn't work well with resque-scheduler [out of the box](https://github.com/JustinAiken/active_scheduler)?

(This change is fine as is, anyway.)

Reply:
Yeah, it's annoying that you need to either add very complex arguments to the resque-scheduler file, or add a package to do it for you.

Is leaving `check_queue` as a rake task okay for now, or did you want me to move it into resque-scheduler?

Reply:
If you don't mind, please and thanks! [AO3-5794](https://otwarchive.atlassian.net/browse/AO3-5794) still has the right cron schedule.

Reply:
I wasn't sure whether you also wanted me to move the code into a second `ActiveJob`, but I can if you want.

Reply:
Would that run into resque-scheduler compatibility issues?

Reply:
Yes, I think I'd have to add a gem or use Rails internal classes to get it to work.

Reply:
Nah, it's not worth it. This is good as is.